### ISSUE TITLE
Travis build broken by old Sys-Virt-0.9.8

### ISSUE THREAD

**Issue by Cyclenerd** [munin-monitoring/contrib#852]
Sys-Virt-0.9.8 does not exist anymore. Oldest version is 1.3.0.

```
--> Working on DANBERR/Sys-Virt-0.9.8.tar.gz
Fetching http://www.cpan.org/authors/id/D/DA/DANBERR/Sys-Virt-0.9.8.tar.gz ... FAIL
! Download http://www.cpan.org/authors/id/D/DA/DANBERR/Sys-Virt-0.9.8.tar.gz failed. Retrying ... 
! Download http://www.cpan.org/authors/id/D/DA/DANBERR/Sys-Virt-0.9.8.tar.gz failed. Retrying ... 
! Download http://www.cpan.org/authors/id/D/DA/DANBERR/Sys-Virt-0.9.8.tar.gz failed. Retrying ... 
! Failed to download http://www.cpan.org/authors/id/D/DA/DANBERR/Sys-Virt-0.9.8.tar.gz
! Failed to fetch distribution Sys-Virt-0.9.8
The command "cpanm --notest DANBERR/Sys-Virt-0.9.8.tar.gz" failed and exited with 1 during .
```
On the Travis server should be `libvirt-dev` 1.2.22 (https://packages.ubuntu.com/trusty/libvirt-dev).

I immediately make a pull request to fix but can not test it.

**Comment by daftaupe** [comment]
Should this be worked on ?
@Cyclenerd, I see in the pull request that you did upload Sys-Virt on your server, but it seems that the link is dead now.
It seems it can be found there : http://lutung.lib.ums.ac.id/cpan/authors/id/D/DA/DANBERR/Sys-Virt-0.9.8.tar.gz

**Comment by Cyclenerd** [comment]
Hi,

https://www.nkn-it.de/backup/sys-virt/Sys-Virt-0.9.8.tar.gz is not deleted. I can also load the file from there.

**Comment by daftaupe** [comment]
Weird, it didn't work earlier for me. 
Anyway, I just show that the build didn't complete on my last PR and I saw this was fixing the thing but wasn't closed.
What's blocking then ?

**Comment by sumpfralle** [comment]
Fixed in 3706fe0051ffa81305938bbfa63df70859338efc
